Models/Website/Npc.php

```
@@ -77,6 +77,41 @@ public function jsonSerialize() : mixed
+                'SELECT COALESCE(MAX(sorting_order), 0) AS max_order FROM npc_quest WHERE quest_id = ?;',
+                array($questId)
+            );
+            $sortingOrder = ($maxOrder !== false ? $maxOrder['max_order'] : 0) + 1;
+
+            $db->executeQuery(
+                'INSERT INTO npc_quest (quest_id, npc_id, sorting_order) VALUES (?, ?, ?);',
+                array($questId, $npcId, $sortingOrder)
+            );
+        }
+
+        return $npcId;
```

PHPStan (2.1.51)

[error] Method VoicesOfWynn\Models\Website\Npc::create() should return int but returns bool|string.


(return.type)